W-10648006 : Adding additional information to the execute command (#1798)
mulesoft/docs-connectors: mongodb/6.3/modules/ROOT/pages/mongodb-connector-reference.adoc

Executes a command on the database.

> The *Execute Command* operation is not a paginated operation, as `executeCommand` returns only results with a single invocation.

Reviewer: ```suggestion
The *Execute Command* operation results are not paginated.
```

> If an execution returns a cursor reference, which contains results with additional elements, the transversal of the results of the command execution must be done by invoking an additional `getMore` command.

Reviewer: ```suggestion
If an execution returns a cursor reference, use the `getMore` command to retrieve the next page of results. 
```
